$NetBSD: patch-apps_tqsl_cpp,v 1.1 2013/09/13 08:45:52 mef Exp $

Avoid following error (interim, only valid with db5)
/usr/pkgsrc/wip/trustedQSL/work/tqsl-2.0/apps/tqsl.cpp:1133:36: error: 'DB_VERSION_STRING' was not declared in this scope
gmake[2]: *** [apps/CMakeFiles/tqsl.dir/tqsl.cpp.o] Error 1
gmake[1]: *** [apps/CMakeFiles/tqsl.dir/all] Error 2

--- apps/tqsl.cpp.orig	2013-08-18 22:07:48.000000000 +0900
+++ apps/tqsl.cpp	2013-09-01 00:39:30.000000000 +0900
@@ -71,7 +71,7 @@
 #endif
 #include <zlib.h>
 #include <openssl/opensslv.h> // only for version info!
-#include <db.h> //only for version info!
+#include <db5/db.h> //only for version info!
 #include "tqslwiz.h"
 #include "qsodatadialog.h"
 #include "tqslerrno.h"
